### PR TITLE
Styling adjustments

### PR DESCRIPTION
**Pull request by hsab**
Use less rounded font
Light yellow color
Screenshot_20200528_162514

### CODE REVIEW

File: src/Containers/Artist/style.module.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 .container {
   margin-bottom: 7vh;
   @include portrait {
-    margin-bottom: mb();
+    margin-bottom: calc(100vw/11);
```
Comment (hsab) [inline src/Containers/Artist/style.module.scss:7]: @lmccart Lauren is this ok?
Fixing the following:
Screenshot_20200528_171354

to:
Screenshot_20200528_171548

File: public/fonts/ariblur/ariblur.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (hsab) [inline public/fonts/ariblur/ariblur.css]: This file can be removed @blerchin

**Comment by blerchin** [comment]
hey @lmccart can you take a look at these changes?
Hirad asked me to take a look at the bugs on Berfin's video and Graham's page. And it seemed like the only way to fix it and avoid writing even more convoluted CSS was to refactor some sections of the ProjectCover.
Also I got consensus that it was ok to make the project titles scroll off the page, which changed things somewhat.

**Review by blerchin — APPROVED** [review]
pretty sure this is good now.

**Pull request merged**